New doc: Manage clusters (#886)
astronomer/docs: astro/view-clusters.md

---
sidebar_label: "View clusters"
title: "View Astro clusters"
id: view-clusters
description: View information about clusters in the Cloud UI.
---

> Use the Cloud UI to get an overview of your Organization's clusters and retrieve cluster information for use by Astronomer support.

Reviewer: ```suggestion
Use the Cloud UI to get an overview of your Organization's clusters and retrieve cluster information that may be required by Astronomer support.

For instructions on how to make a change to an existing cluster, see [Modify a Cluster](modify-cluster.md).
```

> ## View clusters

Reviewer: Any reason to not have a screenshot here?

Author: Not sure a screenshot of the table would help users understand this content any better. We could maybe add a screenshot just of where the tab is, but I think it's more sustainable and accessible to use text for that

In the Cloud UI, click the **Clusters** tab to view information about the clusters in your Organization. The following table provides descriptions for each cluster value.

| Value              | Description                                                                                     |
| ------------------ | ---------------------------------------------------------------------------------------------- |
| Name               | The name of the cluster                                                                        |

> | Provider           | The type of cloud that the cluster is hosted on                                                |

Reviewer: ```suggestion
| Provider           | The cloud provider that the cluster is hosted on. This can be AWS or GCP.                                                |
```

> | Account ID         | The ID of the dedicated Astronomer-managed cloud account for the cluster                      |

Reviewer: ```suggestion
| Account ID         | The ID of the dedicated cloud provider account that the cluster is hosted on and that Astronomer manages. This is an AWS account ID or a Google Cloud project ID.                      |
```

> | Region             | The region where the cluster is hosted in                                                           |

Reviewer: ```suggestion
| Region             | The cloud provider region where the cluster is hosted in                                                          |
```

| VPC subnet         | The range of IP addresses that can be used to connect to the cluster's VPC                    |
| Pod subnet         | GCP only. The range of IP addresses that can be used to connect to the cluster's Pods         |
| Service subnet     | GCP only. The range of IP addresses that can be used to connect to the cluster's GCP services |
| Service peering    | GCP only. The range of IP addresses that can be used to peer with the cluster's VPC           |

> | DB instance type   | The type of instance used for the cluster's metadata database                                  |

Reviewer: ```suggestion
| DB instance type   | The type of instance used for the cluster's primary database                                  |
```

> | Node instance type | The type of instance used for the cluster's worker nodes                                       |

Reviewer: ```suggestion
| Node instance type | The instance type used for the cluster's worker nodes                                       |
```

> | Max node count     | The maximum number of worker nodes supported across all Deployments in the cluster.               |

Reviewer: ```suggestion
| Max node count     | The maximum number of worker nodes supported across all Deployments in the cluster               |
```

> | Updated            | The last time the cluster's settings were updated                                                |

Reviewer: ```suggestion
| Updated            | The date and time the cluster's settings were last updated                                                |
```

> | Created            | When the cluster was created                                                                   |

Reviewer: ```suggestion
| Created            | The date and time at which the cluster was created                                                                   |
```

> For more information about each value, see [AWS resource reference](https://docs.astronomer.io/astro/resource-reference-aws) and [GCP resource reference](https://docs.astronomer.io/astro/resource-reference-gcp)

Reviewer: ```suggestion
 All users in your Organization have access to this view. For more information about each value, see [AWS resource reference](https://docs.astronomer.io/astro/resource-reference-aws) and [GCP resource reference](https://docs.astronomer.io/astro/resource-reference-gcp).
```
